<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addIncomeBox</class>
 <widget class="QDialog" name="addIncomeBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Budget -&gt; Income</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="addIncomeTitle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>360</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Add Income</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>381</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="8" column="0">
     <widget class="QComboBox" name="memberBox">
      <property name="placeholderText">
       <string>Select Member</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="memberLabel">
      <property name="text">
       <string>Member</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="buddy">
       <cstring>categoryBox</cstring>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLabel" name="amountLabel">
      <property name="text">
       <string>Amount</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDateEdit" name="dateEdit">
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2022</year>
        <month>7</month>
        <day>14</day>
       </datetime>
      </property>
      <property name="displayFormat">
       <string>MM/dd/yy</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
      <property name="date">
       <date>
        <year>2022</year>
        <month>7</month>
        <day>14</day>
       </date>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="dateLabel">
      <property name="text">
       <string>Date</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QDoubleSpinBox" name="amountBox">
      <property name="decimals">
       <number>2</number>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="descLabel">
      <property name="text">
       <string>Description</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="buddy">
       <cstring>categoryBox</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="categoryLabel">
      <property name="text">
       <string>Category</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="buddy">
       <cstring>categoryBox</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QComboBox" name="categoryBox">
      <property name="placeholderText">
       <string>Select Category</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="descEdit">
      <property name="placeholderText">
       <string>Enter Description</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="typeLabel">
      <property name="text">
       <string>Type</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="buddy">
       <cstring>categoryBox</cstring>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QComboBox" name="typeBox">
      <property name="placeholderText">
       <string>Select Type</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>addIncomeBox</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>addIncomeBox</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
